.github/workflows/create-channel.yml

```diff
@@ -124,3 +123,35 @@ jobs:
       run: |
         export IMAGE_TAG=${{ steps.tag.outputs.release_tag }}
         docker push ${{ secrets.AWS_ECR_REPO }}/aws-crt-${{ matrix.variant }}:$IMAGE_TAG
+
+  # The job will directly pull the image from the latest, and push to the new tag.
+  # This job is only for a temporary fix. The cheated image would not get updated.
+  cheated-images:
```

Comment:
cheated sounds awkward? maybe something like passthrough image? since we are not generating new image